spot-setup/README.md

```diff
@@ -28,6 +32,11 @@ This file also contains sources desired to be installed as part of Apache Spot,
 
 To read more about these variables, please review the [wiki] (https://github.com/Open-Network-Insight/open-network-insight/wiki/Edit%20Solution%20Configuration).
 
+## Install
+
+1. run `sudo ./install.sh`
+2. run `hdfs_setup.sh`
```

Comment:
Wondering if it's possible to automatize the step of moving `spot.conf` to /etc/ folder in [Spot Installation Documentation, step 3.4](http://spot.incubator.apache.org/doc/).

Also, about your point for discussion, this PR assumes then that everything including spot-ml, spot-ingest and spot-oa is going to be landed in a single node, am I correct?